### ISSUE TITLE
Crash while trying to launch game.

### ISSUE THREAD

**Issue by Emperor-Moon** [YUNG-GANG/YUNGs-Better-Fortresses#12]
I've been getting this crash over and over again that says "The game crashed whilst initializing game
Error: java.lang.RuntimeException: Could not execute entrypoint stage 'main' due to errors, provided by 'betterfortresses'!" I have no idea on what to do since i've been able to launch before with very little problems. Now instead of just 1 crash then being able to launch after with no problem at all, I have tried to launch 47 times and still this error pops up.

**Comment by yungnickyoung** [comment]
Please share the entire latest.log. It's likely that you're using the wrong version of the mod, e.g. trying to load the Forge version on Fabric or vice versa. It's also possible you're using the wrong MC version. Keep in mind my mods are not available on 1.20.2 and 1.20.3.

**Comment by Emperor-Moon** [comment]
Are you refering to the lastest crash log as the log you want?
    On Monday, March 18, 2024 at 07:31:09 PM EDT, Nick Young ***@***.***> wrote:  
 
 


Please share the entire latest.log. It's likely that you're using the wrong version of the mod, e.g. trying to load the Forge version on Fabric or vice versa. It's also possible you're using the wrong MC version. Keep in mind my mods are not available on 1.20.2 and 1.20.3.

**Comment by yungnickyoung** [comment]
Yes, recreate the issue and then it should generate a crashlog. It should also generate a log file called `latest.log`. Being able to view both of those would help me find the issue.

**Comment by williambl** [comment]
Closing as it's been over two years. If the problem is still present for you please send your full logs and we can re-open the issue.
